examples/pytorch/gin/train.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torch.nn.functional as F
+from torch.utils.data.sampler import SubsetRandomSampler
+from sklearn.model_selection import StratifiedKFold
+from dgl.data import GINDataset
+from dgl.dataloading import GraphDataLoader
+from dgl.nn.pytorch.conv import GINConv
+from dgl.nn.pytorch.glob import SumPooling
+import argparse
+
+class MLP(nn.Module):
+    """Construct two-layer MLP-type aggreator for GIN model"""
+    def __init__(self, input_dim, hidden_dim, output_dim):
+        super().__init__()
+        self.linears = torch.nn.ModuleList()
+        # two-layer MLP    
+        self.linears.append(nn.Linear(input_dim, hidden_dim, bias=False))
+        self.linears.append(nn.Linear(hidden_dim, output_dim, bias=False))
+        self.batch_norm = nn.BatchNorm1d((hidden_dim))
+
+    def forward(self, x):
+        h = x
+        h = F.relu(self.batch_norm(self.linears[0](h)))
+        return self.linears[1](h)
+    
+class GIN(nn.Module):
+    def __init__(self, input_dim, hidden_dim, output_dim):
+        super().__init__()
```

Comment:
You can first specify `self.num_layers=5`, which makes the code more readable.

Reply:
Good point. Thanks. I will update accordingly.